src/mono/wasm/runtime/scheduling.ts

```diff
@@ -58,21 +48,18 @@ export function prevent_timer_throttling(): void {
 
 export function schedule_background_exec(): void {
     ++pump_count;
-    if (typeof globalThis.setTimeout === "function") {
-        globalThis.setTimeout(pump_message, 0);
-    }
+    globalThis.setTimeout(pump_message, 0);
 }
 
+let lastSchedule: any = undefined;
```

Comment:
I would prefer a name ending with "id", as `setTimeout` returns a timer/timeout id.
For me `lastTimeoutId` or `lastScheduledTimeoutId` works.